### ISSUE TITLE
Initial Map Position attribute does not support comma as decimal separator

### ISSUE THREAD

**Issue by KaijuBites** [jeffreykemp/jk64-plugin-simplemap#21]
I can only use whole number in Initial Map Position.
![imagen](https://user-images.githubusercontent.com/5175901/47168128-7b674580-d2d6-11e8-806c-cfcf6638082c.png)

I seems that it depends on the values you got for nls_numeric_characters. 
Using
show parameter nls_numeric_characters;
gives me
nls_numeric_characters               string   ,.
Because of that the following code breaks
    l_lat := to_number(substr(l_latlong,1,instr(l_latlong,',')-1));
    l_lng := to_number(substr(l_latlong,instr(l_latlong,',')+1));

It is because to_number expects an ',' and not a '.' as decimal seperator on my system.
I might just change the code to
    l_lat := to_number(substr(l_latlong,1,instr(l_latlong,';')-1));
    l_lng := to_number(substr(l_latlong,instr(l_latlong,';')+1));
Then I should be able to enter the values as follows:
-25,23414;-53,12341243

I guess you could make the decimal seperator and the seperator for longitude and latitude configurable. Or at least mention this limitation in the wiki (if it already is written there, I did not see it).
Thanks for the plugin!

Regards

Nils Stritzel

**Comment by jeffreykemp** [comment]
Hi Nils,

Thanks for raising this. I will look into whether using an alternative delimiter for the two numbers is a suitable solution - I suspect it would be.

In the meantime, another workaround is to use the period (.) as the decimal point, e.g.

-25.23414,-53.12341243

Cheers

**Comment by KaijuBites** [comment]
I guess the work around does not work, because that is how I started as I just copied a position I had gotten from google maps. Actually I don't mind putting a dot as decimal separator, but if I put -25.23414,-53.12341243 I get an error which I traced down to to_number.
On my system if I do the following 

 to_number('-25.0');

 I get ORA-06502: PL/SQL: error : error de conversión de carácter a número numérico o de valor.

With  
to_number('-25,0');
it works though. This is because it depends on the configuration of nls_numeric_characters.
Obviously I cannot insert the values like this -25,23414,-53,12341243 because the string is split on ','.
Anyway I just changed the following to lines of the plugin:
l_lat := to_number(substr(l_latlong,1,instr(l_latlong,';')-1));
l_lng := to_number(substr(l_latlong,instr(l_latlong,';')+1));

Now I can use the comma as decimal separator (which I have to due to the system configuration) as the string is split on the ';': -25,23414;-53,12341243
I hope this clarifies the issue a bit.

Thanks a lot for your efforts.

Cheers
